TACC-Cloud/hazmapper
# DES-1676 - Add ability to update project information in manage panel..

## Overview: ##
Add ability to update project information (name and description) in the Manage panel.

## PR Status: ##


## Related Jira tickets: ##

* [DES-1676](https://jira.tacc.utexas.edu/browse/DES-1676)

## Summary of Changes: ##
Changed `users-panel.component.ts` to implement UI for changing project data.
Changed `projects.service.ts` to make additional calls to the backend to update project data.

## Testing Steps: ##
1. Go to the manage panel in the hazmapper app.
2. Double click or press the edit icon on either name or description.
3. When done, click out of the input or click the check icon.
4. Confirm that the UI is updated.
5. Refresh the browser.
6. Go back to the manage panel.
7. Confirm that the change persisted.

## UI Photos:
![Screenshot from 2020-10-23 12-52-20](https://i.imgur.com/68ro5l9.png)
![Screenshot from 2020-10-23 12-52-20](https://i.imgur.com/cuTyMfU.png)
![Screenshot from 2020-10-23 12-39-48](https://i.imgur.com/WxorFc5.png)


## Notes: ##
Needs some refactoring for duplicate functionality. And more brushing up on UI.
Depends on TACC-Cloud/geoapi#39
